modules/gitops-deploy-argocd-application-in-autonomous-mode.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assembly:
+//
+// * argocd-agent-installation/gitops-deploy-argocd-application-in-autonomous-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-deploy-argocd-application-in-autonomous-mode_{context}"]
+= Deploying an Argo CD application in autonomous mode
+
+In autonomous agent mode, the *spoke* acts as the source of truth. The Argo CD applications must be created directly in the spoke cluster, and the agent reports the application state back to the Principal component in the hub.
+
+.Procedure
+
+. Create the Argo CD application manifest file, `application.yaml`, with the following configuration:
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+name: my-demo-app
+namespace: openshift-gitops # Namespace hosting the agent in the spoke
+spec:
+project: default
+source:
+repoURL: https://github.com/sample-developer/gitops-example-getting-started
```

Comment:
Not a real repo URL?

Reply:
Dhruv use this repo "https://github.com/redhat-developer/openshift-gitops-getting-started"